src/client/app/modules/markupControllers/comment.js

```diff
@@ -6,6 +6,30 @@
       this.defineModels(model);
       this.defineMarkup();
       this.setEventsForModalHandling();
+      this.characterLimit = 450;
+      this.$characterLimitCounterSpan.html(this.characterLimit);
+      
+      this.$target.on("change paste keyup", () => {
+        // limit possible text to the character limit, always
+        this.$target.val(this.$target.val().substr(0, this.characterLimit));
```

Comment:
Please separate this into a function that describes the comment you placed above.